yarn.lock

```diff
@@ -1004,6 +1004,13 @@
   dependencies:
     regenerator-runtime "^0.13.4"
 
+"@babel/runtime@^7.1.2", "@babel/runtime@^7.12.1", "@babel/runtime@^7.4.2", "@babel/runtime@^7.5.5", "@babel/runtime@^7.6.3", "@babel/runtime@^7.7.6":
```

Comment:
oh lol I prob should have put this in the readme, but I've been using NPM for builds instead of yarn (idk why tbh, just... did). So this file isn't being used. I oughta delete it. I'll update readme and stuff later. For now this is fine you don't have to worry about it